modules/oe_content_publication/config/install/field.field.node.oe_publication.oe_publication_collection.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_publication_collection
+    - node.type.oe_publication
+id: node.oe_publication.oe_publication_collection
+field_name: oe_publication_collection
+entity_type: node
+bundle: oe_publication
+label: 'This is a publication collection'
+description: ''
+required: true
```

Comment:
This field need to have a value always.

Reply:
It should not be required. As requested in the ticket, this field should use a checkbox widget in the form display. So, instead of using `options_buttons`, `boolean_checkbox` widget type should be used so we don't force this field to be required.